.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": [
+        true,
```

Comment:
please, your template and original Figma template are not same. Follow Figma styling for your template.

Make change to your code and re request for review.
Thanks.